modules/service/src/main/scala/lucuma/odb/service/ObservationService.scala

```diff
@@ -713,21 +721,24 @@ object ObservationService {
               case Some(lst) if lst.forall(_.status === ConfigurationRequest.Status.Denied) => m.add(ObservationValidation.configuration(ConfigurationRequestMsg.Denied))
               case _ => m.add(ObservationValidation.configuration(ConfigurationRequestMsg.Pending))
 
-        val initialMap = 
-          for {
-            (genVals, op) <- generatorValidations
-            cfpVals       <- cfpValidations
-            itcVals       <- op.filter(_ => cfpVals.isEmpty).foldMapM(itcValidations) // only compute this if cfp and gen are ok
-            bandVals      <- validateScienceBand
-          } yield (genVals |+| itcVals |+| cfpVals |+| bandVals)
+        obsInfo.flatMap: info =>
+          if (info.role.isDefined) List.empty.pure // it's a calibration
```

Comment:
I still wonder about no warnings. Calibrations can be ill defined, e.g. no guide star

Reply:
Hmm. Even if they are automatically generated? And if so, what can the user do about that? Who should be notified and how?

Reply:
Maybe a question for user?

Reply:
Andy commented on the shortcut ticket:

> That's a good question. I am optimistically hopeful that all of our automatically generated calibration observations will be perfect. I could imagine that if all targets were accidentally deleted from the Specphot standard storage program, or a target was added without an SED or brightness information those could cause trouble, but we probably don't want to bother users about that.

> I don't know if we need to solve this now, but we may need a way to give a warning to ourselves if an automatic calibration fails to generate